tests/model_registry/rest_api/conftest.py

```diff
@@ -346,3 +346,27 @@ def model_data_for_test() -> Generator[dict[str, Any], None, None]:
     model_data = copy.deepcopy(MODEL_REGISTER_DATA)
     model_data["register_model_data"]["name"] = model_name
     yield model_data
+
+
+@pytest.fixture()
+def skip_if_not_default_db(request):
+    """
+    Fixture that skips the test if not using default postgres database
+    """
+    default_db = request.node.callspec.params.get("model_registry_metadata_db_resources", {}).get("db_name")
+    LOGGER.info(f"default_db: {default_db}")
+    if not default_db or default_db != "default":
+        pytest.skip(reason="This test is only relevant for default postgres db")
```

Comment:
why not using `@pytest.mark.skipif(condition)` marker and applying it in the test where is needed?